### ISSUE TITLE
A very simple question, if China is not the source of the new crown virus, are you willing to apologize for all the blackness of China on the new crown virus?

### ISSUE THREAD

**Issue by hanshou101** [cirosantilli/china-dictatorship#162]
A very simple question, if China is not the source of the new crown virus, are you willing to apologize for all the blackness of China on the new crown virus?

Looking forward to yours, keep yourself away from double standard answers.

**Comment by cirosantilli** [comment]
I don't understand the question.

**Comment by primeNumberAndMe** [comment]
time will tell the truth

**Comment by MartinX3** [comment]
Yeah, then they will finally believe, that the CCP just does propaganda and that the origin of COVID-19 is in china.

**Comment by ultramax** [comment]
> 
> 
> Hi hanshou101,
> 
> > A very simple question, if China is not the source of the new crown virus, are you willing to apologize for all the blackness of China on the new crown virus?
> 
> I always acknowledge when I discover I'm wrong, and always say what I believe to be the truth very clearly, including when it is good for the CCP.
> 
> See also: https://github.com/cirosantilli/china-dictatorship/tree/303aeb37bdb0e55ef3587e32ee2b849732fde04d#bias

This just proves that you are such a hypocrite. There is literally no scientific evidence that can proves the virus originates in China. You simply want to believe China is the source because you either hate China or you hate CCP or simply both. You tried so hard to make yourself look resourceful and creditable by citing multiple news as your source but at the end of the day it really doesn't matter because your sources are biased against China anyway. It's like Trump supporters try to prove how great Trump is by citing Fox News. Sorry to inform you, this does not work. There are also multiple sources now can prove the virus can be found as early as March 12, 2019 in Spain. Here is just one, https://www.nytimes.com/2020/06/26/health/coronavirus-spain.html. 
If you really "always acknowledge when I discover I'm wrong", you should have apologized a long time ago until there are hardcore scientific evidence that can prove China is the source of the virus
